databases/week3/homework_queries.sql

```diff
@@ -0,0 +1,95 @@
+--Meal Queries:
+--Get all meals
+SELECT * FROM meal;
+--Add a new meal
+INSERT INTO meal(`id`, `title`, `description`, `location`, `when`, `max_reservations`, `price`, `created_date`)
+VALUES(5, 'Bread', 'Garlic Bread', 'Amager', '2021-08-05 12:00:00', 6, 60.30, '2021-07-01 09:00:00');
+--Get a meal with any id, fx 1
+SELECT * FROM meal WHERE id = 3;
+--Update a meal with any id, fx 1. Update any attribute fx the title or multiple attributes
+UPDATE meal
+SET title = 'Waffles', location = 'Vesterbro'
+WHERE id = 5;
+UPDATE meal
```

Comment:
Instead of doing two queries, you can include 'description = 'Nutella & Banana' in the previous query as you did with location

Reply:
Yeah- I changed it to a single query